recipe/patches/0007-fix-export-cxx-std.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/CMake/AbseilDll.cmake b/CMake/AbseilDll.cmake
```

Comment:
Could you apply this on top of the branch that I posted further up and then regenerate the patches with `git format-patch tags/20230125.0`?

Reply:
Done and opened PR at https://github.com/h-vetinari/abseil-cpp/pull/1 .

Reply:
Thanks a lot; though once it's in a format that `git am` understands, I don't need the PR actually, can just pick it up from the recipe 👍